Library/Homebrew/cmd/update.rb

```diff
@@ -41,8 +41,10 @@ def update
     master_updater.pull!
     master_updated = master_updater.updated?
     if master_updated
-      puts "Updated Homebrew from #{master_updater.initial_revision[0, 8]} " \
-           "to #{master_updater.current_revision[0, 8]}."
+      initial_revision_short = shortenRevision(master_updater.initial_revision)
+      current_revision_short = shortenRevision(master_updater.current_revision)
+      puts "Updated Homebrew from #{initial_revision_short} " \
+           "to #{current_revision_short}."
```

Comment:
Thanks for the tip, I wasn't too familiar with the style conventions, so I just took a guess. Addressed in https://github.com/Homebrew/homebrew/commit/32bf16e4211ac1df254e2b973ad21fd3fc53f6fd